xlators/storage/posix/src/posix-helpers.c

```diff
@@ -376,6 +387,8 @@ _posix_get_marker_all_contributions(posix_xattr_filler_t *filler)
     ret = 0;
 
 out:
+    if (dyn_memory)
+        GF_FREE(list);
```

Comment:
if the memory allocation in line 356 fails, will this GF_FREE() succeed?

Reply:
yes it will be succeeded because list will be NULL in case if memory allocation is failed, there is no impact if we do pass
NULL to GF_FREE.